src/adapter/debugAdapter.ts

```diff
@@ -262,6 +264,48 @@ export class DebugAdapter {
     return {};
   }
 
+  async _toggleSkipFileStatus(params: Dap.ToggleSkipFileStatusParams): Promise<Dap.ToggleSkipFileStatusResult> {
+    if (params.resource) {
+      const absPath = urlUtils.fileUrlToAbsolutePath(params.resource);
+      if (absPath) {
+        params.resource = absPath;
+      }
+    }
+
+    if (!this.sourceContainer.scriptSkipper) {
+      if (!this._thread || !this._thread.cdp()) {
+        return {};
```

Comment:
Not clear to me why this is lazily initialized. Seems like some code would be simpler if was created immediately on startup